60-encapsulation.md

```diff
@@ -0,0 +1,36 @@
+# Define encapsulation in Python
+
+`Encapsulation` - Restrict the access to methods or variables. This can prevent the data from being modified or accessing accidentally, but not intentionally.
+
+The private attributes and methods are not really hidden, they’re renamed adding `_name` in the beginning of their name. The method can actually be called.
+
+`Example:`
+
+We create a class `Car` which has two methods:  `drive()` and `updateSoftware()`.  When a `car` object is created, it will call the private methods `__updateSoftware()`.  
```

Comment:
```suggestion
We create a class `Car` which has two methods:  `drive()` and `updateSoftware()`. When a `car` object is created, it will call the private methods `__updateSoftware()`.  
```